.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
Sorry, I should been clearer. The only thing I'm worried about is preserving existing functionality of reading testlets and collectors from the filesystem. Currently the todd-server will unpack assets and write them to the filesystem once, on startup. Then it will start to serve all files in that directory. I'm even fine with removing the "unpacking" part, and serving the native testlets/collectors from memory without writing them out to the FS before serving - but I want to preserve that bit that allows someone to pop a new one in the directory on the server and have it get served once todd-server is restarted.

In short, I want to be able to serve up testlets/collectors from both locations, and the default ones will just be kept in memory. Also, the concept of being able to add a new testlet "hot" (i.e. no todd-server restart) is not currently implemented and I'm not expecting that this PR does that. I've updated https://github.com/Mierdin/todd/issues/27 to reflect that this kind of thing should be considered later.